php/blocks/block-coauthor-avatar/class-block-coauthor-avatar.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Co-Author Avatar
+ *
+ * @package Automattic\CoAuthorsPlus
+ * @since 3.6.0
+ */
+
+namespace CoAuthors\Blocks; 
+
+use WP_Block;
+/**
+ * Block CoAuthor Avatar
+ *
+ * @package CoAuthors
+ */
+class Block_CoAuthor_Avatar {
+
+	/**
+	 * Register Block
+	 *
+	 * @since 3.6.0
+	 */
+	public static function register_block(): void {
+		register_block_type(
+			dirname( COAUTHORS_PLUS_FILE ) . '/build/blocks/block-coauthor-avatar',
+			array(
+				'render_callback' => array( __CLASS__, 'render_block' ),
+			)
+		);
+	}
+	/**
+	 * Render Block
+	 *
+	 * @since 3.6.0
+	 * @param array    $attributes
+	 * @param string   $content
+	 * @param WP_Block $block
+	 * @return string
+	 */
+	public static function render_block( array $attributes, string $content, WP_Block $block ): string {
```

Comment:
@GaryJones Same PHP compatibility question [as below](https://github.com/Automattic/Co-Authors-Plus/pull/997/files#r1364511842) for the scalar and return value type hinting. Should these be modified to be PHP 5.6 compatible?